taxonomies/categories.txt

```diff
@@ -98317,6 +98317,7 @@ ru:Непищевые продукты, Не еда
 en:Open Pet Food Facts, OPFF
 es:Open Pet Food Facts, OPFF
 fr:Open Pet Food Facts, OPFF
+hr:Open Pet Food Facts, OPFF
```

Comment:
all this should be changed to:

en:Open Pet Food Facts, OPFF
xx:Open Pet Food Facts, OPFF

The xx: prefix will match all languages.

Same for other entries where the translation is in fact the same as the English entry.

Reply:
I changed it in #7535 . I happened to add something

Reply:
There are too much conflicts now.

For example, are you sure you wanted to remove "en:Open Products Facts, OPF" and all its children in this PR: https://github.com/openfoodfacts/openfoodfacts-server/pull/3639

Reply:
@stephanegigandet 
I can cancel this PR and restart from actual main branch. 

But before that, can you confirm that removing "en:Open Products Facts, OPF" and all its children in this other PR: https://github.com/openfoodfacts/openfoodfacts-server/pull/3639 was expected? 
Because this other PR was opened for some time, and I wonder if this deletion was not done unintentionally. 
Sorry to instist

Reply:
@benbenben2 I don't know if it was intentional, I'm guessing it was not, as we still have the categories for OBF and OPFF.

Reply:
@benbenben2, I merged with main and fixed conflicts (putting back OPF and keeping additional entries in the main)
Please, review

Reply:
ping @stephanegigandet  (instead of myself :-D )